package.json

```diff
@@ -6,5 +6,8 @@
 	"scripts": {
 		"docker-up": "docker-compose up",
 		"docker-down": "docker-compose down"
+	},
+	"dependencies": {
```

Comment:
You should add this dependencies to the package.json inside the sever folder: "server/package.json"

Reply:
This dependency was added automatically by node when I installed nodemailer. I'll leave it in the archive you mentioned. Also, node modified package-lock.json, is it ok?